pkg/controller/argocd/keycloak.go

```diff
@@ -540,7 +542,7 @@ func createRealmConfig(cfg *keycloakConfig) ([]byte, error) {
 				DisplayName: "Login with OpenShift",
 				ProviderID:  "openshift-v4",
 				Config: map[string]string{
-					"baseUrl":      fmt.Sprintf("https://%s", "kubernetes.default.svc"),
+					"baseUrl":      fmt.Sprintf("https://%s", "kubernetes.default.svc.cluster.local"),
```

Comment:
Why not just "https://kubernetes.default.svc.cluster.local"?

Reply:
acknowledge